public/index.ts

```diff
@@ -859,7 +859,7 @@ function onChannelDataArrived(v: Volume, channelIndex: number) {
 
   currentVol.channels[channelIndex].lutGenerator_percentiles(0.5, 0.998);
   view3D.onVolumeData(currentVol, [channelIndex]);
-  view3D.setVolumeChannelEnabled(currentVol, channelIndex, channelIndex < 3);
+  //view3D.setVolumeChannelEnabled(currentVol, channelIndex, channelIndex < 3);
```

Comment:
removing this line let me turn channels on and off during playback when new data was still arriving.

Reply:
Nit: delete, or add a comment when commenting out a line.

Reply:
Extra nit: Add a TODO for easier searching later?